src/librustc/mir/visit.rs

```diff
@@ -247,42 +247,30 @@ macro_rules! make_mir_visitor {
 
             fn super_mir(&mut self,
                          mir: & $($mutability)* Mir<'tcx>) {
-                let Mir {
```

Comment:
Either a trait implemented for `&mut Mir` and `&Mir` differently or `stringify!($mutability).is_empty()`.